File: python/retangulo.py
import math

x = float(input("Base do retangulo: "))
y = float(input("Altura do retangulo: "))

area = x * y
perimetro = 2 * (x + y)
diagonal = math.sqrt(x ** 2 + y ** 2)

print(f"AREA = {area:.4f}")
print(f"PERIMETRO = {perimetro:.4f}")
print(f"DIAGONAL = {diagonal:.4f}")

File: c/retangulo.c
#include <stdio.h>
#include <math.h>

int main(){

    float base, altura, area, perimetro, diagonal;

    printf("Base do retangulo: ");
    scanf("%f", &base);

    printf("Altura do retangulo: ");
    scanf("%f", &altura);

    area = base * altura;
    perimetro = 2 * (base + altura);
    diagonal = sqrt(pow(base, 2) + pow(altura, 2));

    printf("AREA = %.4f\n", area);
    printf("PERIMETRO = %.4f\n", perimetro);
    printf("DIAGONAL = %.4f\n", diagonal);

    return;
}
